praw/models/reddit/live.py

```diff
@@ -6,95 +6,6 @@
 from .redditor import Redditor
 
 
```

Comment:
line 9-97: Sort class order lexycographically. Please tell me if this change isn't needed.